src/rendering/texformats.jl

```diff
@@ -155,17 +163,22 @@ end
 get_tex_template(::Nothing) = get_template(normpath(TEMPLATE_DIR, "md2pdf.tpl"))
 get_tex_template(x) = get_template(x)
 
-function render_doc(docformat::JMarkdown2tex, body, doc)
+highlight_str(docformat::TexFormat) = ""
+highlight_str(docformat::JMarkdown2tex) =
+    get_highlight_stylesheet(MIME("text/latex"), docformat.highlight_theme)
```

Comment:
Maybe defining `render_doc` for `JMarkdown2tex` and `TexFormat` separately will make more sense ?

Reply:
That is probably true. This saved a handful of lines but of course it needs another new functions.

Reply:
defined `render_doc` separately now